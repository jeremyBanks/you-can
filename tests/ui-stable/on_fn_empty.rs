#[you_can::turn_off_the_borrow_checker]
fn main() {}
