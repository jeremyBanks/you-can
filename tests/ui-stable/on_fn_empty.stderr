
[48;5;9m[38;5;15m[1m[5m DANGER [0m  This project is using the the [1m#[you_can::turn_off_the_borrow_checker][0m
[48;5;0m[38;5;9m[1m[5m DANGER [0m  macro, which is inherently unsafe, unsound, and unstable. This is not
[48;5;15m[38;5;0m[1m[5m DANGER [0m  suitable for any purpose beyond educational experimentation.

